_includes/_blogcontent/2024-06-04-HistoricLit.md

```diff
@@ -12,7 +12,11 @@ Through the mid-20th century, many observatories had their own outlets for disse
  
 For three decades, the [NASA Astrophysics Data System (NASA ADS)](https://ui.adsabs.harvard.edu/about/history/) served as the digital library for astronomy and astrophysics. Consequently, NASA ADS indexed, linked to, and, when allowed, hosted scanned versions of historic observatory publications enabling everyone to find and access their contents. For our initial collection, we partnered with the [John G. Wolbach Library](https://www.cfa.harvard.edu/lib) at the [Center for Astrophysics | Harvard & Smithsonian](https://www.cfa.harvard.edu/). The librarians there and at the [James M. Gillis Library](https://www.cnmoc.usff.navy.mil/Our-Commands/United-States-Naval-Observatory/The-James-M-Gilliss-Library/) at the [US Naval Observatory](https://www.cnmoc.usff.navy.mil/usno/) prioritized scanning observatory publications with complete runs available. The results were stored on 35-mm film, which NASA ADS, in turn, digitized to produce the files we now host on our servers. We entrusted one copy of the films to each library. Now, [NASA SciX](https://scixplorer.org/scixabout/) has assumed responsibility for this important legacy literature collection. 
  
+<<<<<<< AAS-244-HistoricLiterature
+NASA SciX continues to digitize legacy materials as we receive them. Librarians and researchers send us publications as they reduce their collections. Small astronomy organizations have asked us to become their repository. We estimate our historic astronomy literature collection is over 80% complete. We measure completeness by the fraction of papers that are referenced articles in the core astronomy and astrophysics journals for which we have entries.  
+=======
 NASA SciX continues to digitize legacy materials as we receive them. Librarians and researchers send us publications as they weed their collections. Small astronomy organizations have asked us to become their repository. We estimate our historic astronomy literature collection is over 80% complete. We measure completeness by the fraction of papers that are referenced articles in the core astronomy and astrophysics journals for which we have entries.  
+>>>>>>> master
```

Comment:
I think something got funky when you merged some changes? this version has kept both the master and the branch version of this file. Also I cant see the other file changes that you made. Maybe something accidentally merged into master?

Reply:
I only had a single word change for this request. I'm not sure what happened, but it looks like something went wrong when I tried to resolve the conflict before you reviewed it.